src/main/java/com/epam/controller/ToursController.java

```diff
@@ -104,7 +105,7 @@ public ModelAndView addTour(@RequestParam String tourType, @RequestParam String
                     .pricePerUnit(addPricePerPerson)
                     .hotelId(1) //stub
```

Comment:
Why hotelId is still stubbed?

Reply:
I believe we can fix it in another request, this is only about discount, isnt it?

Reply:
yes, only about discount